content/en/about.md

```diff
@@ -0,0 +1,30 @@
+---
+title: About
+---
```

Comment:
Are you sure that you want an about page? If not, drop this page.

Reply:
I think I'll keep it in for now, and have a discussion with the maintainers about it.

I've been looking at the issues/PRs list and can't find the discussion we had with the rationale behind cutting it. I remember agreeing, but it'd be good to link the issue. Do you recall where we had the conversation, @chalin?

For context, @SteveLasker, the whole site is basically _about_ the project, so a specific _about_ page can be a bit redundant (I know I've included an _about_ in the docs assessment IA, but included it as a placeholder for discussion--I should be more clear about that in the assessment itself).

Reply:
FYI: https://github.com/etcd-io/website/issues/434 :)

Reply:
Thanks!